benchmark/run

```diff
@@ -12,17 +12,18 @@ server_addr="$1"
 mkdir -p ./log
 	
 benchmark_siege() {
-	siege -R ./etc/siege.conf -c 8 -r 100000 -q 	\
-		--log=./log/$1.log "$1://$server_addr" 	\
+	siege -R ./etc/siege.conf -c 8 -r 100000 -q	\
```

Comment:
just a suggestion, not for this particular PR though.. can't we move `-c` and `-r` parms to the config file? if not, I would put this in variables in this same file..

Reply:
we could. I isolated them anticipating the need to parameterize these values. the idea was to plot a chart of time x request-per-minute, for example, so we would pass these values as parameter to the script.

in`siege.conf` we could set `concurrent = 8` and `reps = 10000`, for instance.